assisted-events-scrape/events_scrape/logger.py

```diff
@@ -3,57 +3,30 @@
 import os
 import re
 import sys
-
-
-class SensitiveFormatter(logging.Formatter):
-    """Formatter that removes sensitive info."""
-
-    @staticmethod
-    def _filter(s):
-        # Dict filter
-        s = re.sub(r"('_pull_secret':\s+)'(.*?)'", r"\g<1>'*** PULL_SECRET ***'", s)
-        s = re.sub(r"('_ssh_public_key':\s+)'(.*?)'", r"\g<1>'*** SSH_KEY ***'", s)
-        s = re.sub(r"('_vsphere_username':\s+)'(.*?)'", r"\g<1>'*** VSPHERE_USER ***'", s)
-        s = re.sub(r"('_vsphere_password':\s+)'(.*?)'", r"\g<1>'*** VSPHERE_PASSWORD ***'", s)
-
-        # Object filter
-        s = re.sub(r"(pull_secret='[^']*(?=')')", "pull_secret = *** PULL_SECRET ***", s)
-        s = re.sub(r"(ssh_public_key='[^']*(?=')')", "ssh_public_key = *** SSH_KEY ***", s)
-        s = re.sub(r"(vsphere_username='[^']*(?=')')", "vsphere_username = *** VSPHERE_USER ***", s)
-        s = re.sub(r"(vsphere_password='[^']*(?=')')", "vsphere_password = *** VSPHERE_PASSWORD ***", s)
-
-        return s
-
-    def format(self, record):
-        original = logging.Formatter.format(self, record)
-        return self._filter(original)
-
+from pythonjsonlogger import jsonlogger
+
+def get_custom_format() -> str:
+    log_fields = [
+        "asctime",
+        "name",
+        "levelname",
+        "thread",
+        "message",
+        "pathname",
+        "lineno",
+    ]
+    log_format = lambda x: ['%({0:s})s'.format(i) for i in x]
+    return ' '.join(log_format(log_fields))
```

Comment:
maybe this will be a bit more clear:
```suggestion
    return " ".join(f"%({field})s" for field in log_fields)
```